pwnlib/args.py

```diff
@@ -159,6 +159,10 @@ def STDERR(v):
     """Sends logging to ``stderr`` by default, instead of ``stdout``"""
     context.log_console = sys.stderr
 
+def LOCAL_LIBCDB(v):
+    """Sets local libcdb path"""
```

Comment:
We should mention `context.local_libcdb` being modified here and what it is about.